ballerina/build.gradle

```diff
@@ -297,9 +315,16 @@ task packageDistZip(type: Zip) {
     }
 
     /* Standard Libraries */
+    def birCache = new File("${parentDir}/distributions/jballerina-${ballerinaVersion}/cache/bir/")
+    def baloCache = new File("${parentDir}/distributions/jballerina-${ballerinaVersion}/cache/balo/")
+    def jarCache = new File("${parentDir}/distributions/jballerina-${ballerinaVersion}/cache/jar/")
```

Comment:
Isn't it better to have this path (up to `cache`) as a parameter, since we use it multiple places?

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/510/commits/4cac0641cf76edb31555ea7afaa9b6e35f536b8e